app/__init__.py

```diff
@@ -273,6 +274,9 @@ def register_blueprint(application):
     govuk_alerts_blueprint.before_request(requires_govuk_alerts_auth)
     application.register_blueprint(govuk_alerts_blueprint)
 
+    admin_blueprint.before_request(requires_admin_auth)
+    application.register_blueprint(admin_blueprint, url_prefix="/admin")
```

Comment:
Happy to mount this anywhere else.

I don't particularly like the layout of the API as it stands right now .. would love to have a chat around refactoring it. But that's for later. For now the `find-by-uuid` doesn't feel like it fits into any of our existing buckets, so we need a new one. We can either go for a generic 'admin' bucket, or we could have a `lookups` bucket? Or something else?

Reply:
`/platform-admin`? i think `/admin` is super generic, and is immediately unclear about whether it refers to the notify-admin app or the notify team - who should rightfully be the only people who can use this endpoint

Reply:
So, I had mounted it at platform-admin originally as well. However, it felt like a slight misnomer because it will be used by the admin app authenticating as the admin app (because the admin app doesn't pass through the user). So all of the authorization happens in the admin app invisible to the api. Mounting it at `platform-admin` makes me want to apply some permissions checks in the api that assert it's being called by a platform admin, but we can't do that (sidenote: I would like the admin app to pass the acting user through to the api eventually, so that the api can make assertions).